abc.templates/base-workflows/contents/guardian-admin.yml

```diff
@@ -129,11 +127,9 @@ jobs:
         env:
           # used by terraform when using the github provider
           TERRAFORM_COMMAND: '${{ inputs.command }}'
-          TERRAFORM_ARGS: '${{ inputs.args }}'
           DIRECTORY: '${{ matrix.working_directory }}'
         shell: 'bash'
         run: |-
           guardian run \
-          -terraform-command="${TERRAFORM_COMMAND}" \
-          -terraform-args="${TERRAFORM_ARGS}" \
-          "${DIRECTORY}"
+          -dir="${DIRECTORY}" \
+          -- ${TERRAFORM_COMMAND}"
```

Comment:
I did some testing and this should not be subject to any injection type attacks.
For example if the user typed in `; echo 'hello'` then each 'word' would get passed in as guardian command line args. i.e. guardian's args would be `["run", "-dir=directory", "--", ";", "echo", "'hello'"]` which would simply lead to an error when the args  `[";", "echo", "'hello'"]` get passed along to the terraform binary.

Reply:
Great thank you!